pulsar-broker/src/main/java/org/apache/pulsar/PulsarClusterMetadataSetup.java

```diff
@@ -115,6 +115,13 @@
                 description = "The metadata service URI of the existing BookKeeper cluster that you want to use")
         private String existingBkMetadataServiceUri;
 
```

Comment:
This annotation is not really meaningful here by itself, can we add a comment?

Reply:
Which annotation? Do you mean `@Deprecated`?